pagopa/io-app
# [#172880598] Update design of QR scanner screen

**SHORT DESCRIPTION:**
This pr introduces a restyle of the screen used to scan QR codes (for recognizing a payment from an analogic notice). Moreover, it revert strings 'QR code' to 'QR Code'.

**DETAILS ON CHANGES:**
Some changes are introduced to grand the following paragraph is not displayed if the app hasn't the camera permissions.
<img width="306" alt="image" src="https://user-images.githubusercontent.com/38431762/82156214-3d4b5280-987a-11ea-9cc4-aa9c33fd4565.png">
this content is presented as the bottom part of the `QRCodeScanner` component and the component does not implement a check that allows us to hide it if the permissions to the camera are not granted by the user. As a consequence, to properly hide it :
- the screen content is not scrollable anymore
- the height of the camera view is reduced to take the 2/3 of the available space between the header and the footer  (so the user don't need to scroll to read all the content - it was occurring on some android devices)
- the content displayed when the camera permissions are not granted has a default height equal to the screen height
When we haven't the camera permissions, these conditions make the bottom content is rendered outside the displayed area and, having disabled the scroll, the user can't display it.

**SCREEN PREVIEW:**
Here the updated screens:
* the camera usage is disabled (and the user has to enable it from settings):  <img width="500" alt="image" src="https://user-images.githubusercontent.com/38431762/82156118-8bac2180-9879-11ea-91c1-c73614d0e767.png">

* the camera usage is permitted <img width="1222" alt="image" src="https://user-images.githubusercontent.com/38431762/82156127-99fa3d80-9879-11ea-9be3-2baede119dc6.png">

**HOW TO REPRODUCE:**
- go to settings and revoke the camera permissions
- return to app > payment home > click on 'pay notice
- click on ''go to settings" and enable camera permissions
- return to the app (on the same screen below)
